app/scripts/modules/google/src/loadBalancer/configure/http/backendService/backendService.component.html

```diff
@@ -130,7 +128,7 @@
       </div>
     </div>
 
-    <div class="form-group">
+    <div ng-if="!$ctrl.loadBalancer.isInternal" class="form-group">
```

Comment:
This is a little confusing because Spinnaker already supports internal TCP/UDP load balancers, but I'm guessing the `isInternal` here means internal HTTP(S)? Could we check the type explicitly here instead?

Reply:
Yes, this field meas internal HTTP(S). But ok, I'll fix it.